commands/keplr.js

```diff
@@ -281,10 +281,50 @@ const keplr = {
       playwright.keplrWindow(),
       true,
     );
+
+    await playwright.waitForURLLoad(originalURL);
+    await playwright.switchToCypressWindow();
+
+    return true;
+  },
+
+  async addNewTokensFound() {
+    module.exports.switchToKeplrIfNotActive();
+    await module.exports.goToHome();
+    await playwright.waitAndClickByText(homePageElements.newTokensFound);
+    await playwright.waitAndClick(
+      homePageElements.selectAllTokensCheck,
+      await playwright.keplrWindow(),
+      { number: -1, force: true },
+    );
+    await playwright.waitAndClickByText(
+      homePageElements.addChainsButton,
+      await playwright.keplrWindow(),
+      true,
+    );
     await playwright.switchToCypressWindow();
 
     return true;
   },
+
+  async getTokenAmount({ tokenName }) {
+    module.exports.switchToKeplrIfNotActive();
+    await module.exports.goToHome();
+
+    const tokenLabel = await playwright.waitFor(
+      homePageElements.tokenNameLabel(tokenName),
+    );
+    const parentElement = tokenLabel.locator(
+      homePageElements.tokenParentSelector,
+    );
+    const innerTexts = await parentElement.allInnerTexts();
+    const textArray = innerTexts[0].split('\n');
+    const tokenValue = Number(textArray[3]);
+
```

Comment:
Should we add some fallback logic in case the token we're trying to find does not exist?

Reply:
i think this is fine, since it should crash if the expected token is not found. 
but any suggestions?